Tests.Integration.Presentation.Web/Tools/External/Rights/RightsHelper.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Core.DomainModel;
+using Core.DomainModel.Organization;
+using Presentation.Web.Models.API.V1;
+using Tests.Integration.Presentation.Web.Tools.Model;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.Tools.External.Rights
+{
+    public class RightsHelper
+    {
+        public static async Task AddUserRole(int userId, int orgId, RightsType rightsType, string name, Cookie optionalLogin = null)
+        {
+            var cookie = optionalLogin ?? await HttpApi.GetCookieAsync(OrganizationRole.GlobalAdmin);
+            
+            var roleDto = new RightDTO
+            {
+                UserId = userId,
+                RoleId = (int)OrganizationRole.LocalAdmin
```

Comment:
it is called "AddUserRole" but the role is hard coded to "LocalAdmin"?

Reply:
Also OrganizationRole enum is only valid to use for organization roles, not other rights types

Reply:
for the other types the choice types must be used..